pom.xml

```diff
@@ -48,9 +48,9 @@
 
     <main.basedir>${project.basedir}</main.basedir>
 
-    <armeria.version>0.86.0</armeria.version>
+    <armeria.version>0.87.0</armeria.version>
     <!-- This is from armeria, but be careful to avoid >= v20 apis -->
-    <guava.version>27.1-jre</guava.version>
+    <guava.version>28.0-jre</guava.version>
```

Comment:
Don't remember exactly what this is but I wonder if it's removable since we switched to `-protocol`. Armeria itself shades guava

Reply:
you might be right